src/util.F90

```diff
@@ -1970,6 +1970,19 @@ integer function pow2_above(n)
 
   end function pow2_above
 
+!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+  !> Returns the current system clock time in seconds.
+  real(kind=dp) function system_clock_time()
+
+    ! 64 bit integer enables hi-resolution time.
+    integer(kind=8) :: clock_count, clock_count_rate
+
+    call system_clock(clock_count, clock_count_rate)
+    system_clock_time= real(clock_count, kind=dp) / clock_count_rate
```

Comment:
We really need an automatic code formatter!
```suggestion
    system_clock_time = real(clock_count, kind=dp) / clock_count_rate
```

Reply:
> We really need an automatic code formatter!

https://github.com/fortran-lang/fprettify